Libraries/Text/Text.js

```diff
@@ -124,6 +128,9 @@ const Text = React.createClass({
   contextTypes: {
     isInAParentText: React.PropTypes.bool
   },
+  hasPressHandler(): boolean {
```

Comment:
Can you prefix the method with an `_` otherwise it will show up in the docs on the website.